spec/index.bs

```diff
@@ -5205,7 +5222,11 @@ dictionary GPUSamplerDescriptor
         </div>
 </dl>
 
-Issue: explain how LOD is calculated and if there are differences here between platforms.
+<dfn dfn lt="levels of detail|lod">Level of detail</dfn> (LOD) describes which mip level(s) are selected when sampling a texture.
+It may be specified explicitly through shader methods like [=textureSampleLevel=] or implicitly
+determined from the [=texture coordinate=] derivatives. See
+[[vulkan#textures-lod-and-scale-factor|Scale Factor Operation, LOD Operation and Image Level Selection]] in the
+[[vulkan inline]] spec for details on how implicit LODs are calculated.
```

Comment:
This appears to be normative; it shows up under the normative references section: https://gpuweb.github.io/gpuweb/#normative

Reply:
I shouldn't have been so quick to approve it, thanks for noticing

Reply:
Yes, I just noticed that I apparently forgot to `git add` the update before committing and pushing. Sorry! I fixed it, and then pushed the intended fix here: https://github.com/gpuweb/gpuweb/commit/bbe58b845e1465ef6c9e552d18b40d25d401b9e5 (and was mildly surprised that I was able to push directly to main.)

Reply:
Sounds good, thanks for fixing!